index.html

```diff
@@ -6,7 +6,7 @@
     <title>Data Tools</title>
     <link href="https://fonts.googleapis.com/css?family=Sarabun" rel="stylesheet">
     <link href="/dist/index.css" rel="stylesheet">
-    <link rel="shortcut icon" href="https://d2tyb7byn1fef9.cloudfront.net/ibi_group-128x128.png" type="image/x-icon">
+    <link rel="shortcut icon" href="https://d2tyb7byn1fef9.cloudfront.net/ibi-logo-original%402x.png" type="image/x-icon">
```

Comment:
I find this logo to be kind of fuzzy and illegible. What do you think about using this one instead? https://d2tyb7byn1fef9.cloudfront.net/ibi-128x128_bg.png